components/usage/pkg/db/dbtest/workspace.go

```diff
@@ -5,20 +5,74 @@
 package dbtest
 
 import (
+	"fmt"
 	"github.com/gitpod-io/gitpod/usage/pkg/db"
 	"github.com/google/uuid"
+	"math/rand"
 	"testing"
 )
 
-func NewWorkspace(t *testing.T, id string) db.Workspace {
+const (
+	WorkspaceContext = `{"title":"[usage] List workspaces for each workspace instance in usage period","repository":{"cloneUrl":"https://github.com/gitpod-io/gitpod.git","host":"github.com","name":"gitpod","owner":"gitpod-io","private":false},"ref":"mp/usage-list-workspaces","refType":"branch","revision":"586f22ecaeeb3b4796fd92f9ae1ca3512ca1e330","nr":10495,"base":{"repository":{"cloneUrl":"https://github.com/gitpod-io/gitpod.git","host":"github.com","name":"gitpod","owner":"gitpod-io","private":false},"ref":"mp/usage-validate-instances","refType":"branch"},"normalizedContextURL":"https://github.com/gitpod-io/gitpod/pull/10495","checkoutLocation":"gitpod"}`
```

Comment:
Down the line, we can make this a nicer object with helpers to serialize & deserialize. For now, we don't use it in usage so left it as raw JSON